pyproject.toml

```diff
@@ -1,7 +1,8 @@
 [tool.poetry]
-package-mode = false
 name = "evap"
 description = "EvaP"
+authors = [ "EvaP Developers" ]
```

Comment:
Should we add the organization members here? Right now, PyPI displays "EvaP Developers" under unverified details, I suppose we could also just add the names which will hopefully be deduplicated then

Reply:
Looks like it won't be deduplicated, I suppose no one really cares about this, see https://test.pypi.org/project/evap/2024.10.0/ or https://pypi.org/project/black/

Reply:
I must be blind -- what section exactly would contain duplicated names? I see 5 unique user names on the black project page under "Maintainers"

I'd be fine with just putting "EvaP Developers", which will always remain correct.

Do we even need this in the pyproject.toml? I would expect that for username mapping, you need to configure this on PyPI separately anyway?

Reply:
"Ambv" is the same person as the unverified "author" (=> big projects don't care about about unverified data) and also on the evap release my name appears twice (=> it doesn't get deduplicated)

Reply:
Ah, I see -- I was looking for duplication inside one list. Yeah, "evap developers" 👍